radix-engine-interface/src/blueprints/resource/non_fungible/non_fungible_resource_manager.rs

```diff
@@ -420,15 +420,15 @@ impl<'a> Arbitrary<'a> for NonFungibleDataSchema {
     // ScryptoSchema, therefore implementing arbitrary by hand.
     // TODO: Introduce a method that genearates NonFungibleDataSchema in a truly random manner
     fn arbitrary(_u: &mut Unstructured<'a>) -> Result<Self> {
-        Ok(Self::Local {
-            schema: VersionedSchema::V1(SchemaV1 {
+        Ok(Self::Local(LocalNonFungibleDataSchema {
```

Comment:
SIDE NOTE (unimportant): It seems to be that `scrypto_decode(unstructured.bytes()).map_err(...)` might be better than this, but maybe it converges too slowly to a valid value.